db/fixtures/ae_datastore/ManageIQ/System/Event/EmsEvent/RHEVM.class/network_interface_plugged_into_vm.yaml

```diff
@@ -9,4 +9,4 @@ object:
     description: 
   fields:
   - rel4:
-      value: "/System/event_handlers/event_action_refresh?target=ems"
+      value: "/System/event_handlers/event_action_refresh?target=src_vm"
```

Comment:
There is no need to target ems. It is OK to target vm. There is noting specific in this patch which enable it.